### PR TITLE
Add sliding epoch option to matplotlib plot

### PR DESCRIPTION
**Pull request by Bartolo1024**

### CODE REVIEW

**Comment by Bartolo1024** [comment]
@stared I would like to refresh one thing from our discussion. The sliding option should be available as global in the main logger (not matplotlib only as there is)? I remember that you told me about sth like this, but it would be strange to me :|

**Comment by stared** [comment]
Solving #124.

More or less in this part of code, related to the MainLogger.

A few things to do:

* `min(self.start_epoch, len(xs) - self.start_epoch` deserves a name `x_first`, rather than repetition
* there should be a rule for negative  `self.start_epoch`
* in general, a better therm is `from_epoch`(or `first_epoch`); to me `start_epoch` sounds less clear

**Comment by Bartolo1024** [comment]
A negative value of the start_epoch (now from_epoch is handled by the same formula)

File: livelossplot/outputs/matplotlib_plot.py [hunk 1]
```diff
@@ -123,9 +123,11 @@ def _draw_metric_subplot(self, ax: plt.Axes, group_logs: Dict[str, List[LogItem]
         for name, logs in group_logs.items():
             if len(logs) > 0:
                 xs = [log.step for log in logs]
-                xs = xs[min(self.start_epoch, len(xs) - self.start_epoch):]
+                x_first = min(self.from_epoch, len(xs) - self.from_epoch)
+                xs = xs[x_first:]
                 ys = [log.value for log in logs]
-                ys = ys[min(self.start_epoch, len(ys) - self.start_epoch):]
+                y_first = min(self.from_epoch, len(ys) - self.from_epoch)
```
Comment (stared) [inline livelossplot/outputs/matplotlib_plot.py:129]: Is it possible that `y_first != x_first`?

Comment (Bartolo1024) [reply livelossplot/outputs/matplotlib_plot.py]: no, done

**Pull request closed without merging**